content/en/core/releases/4.4.0.md

```diff
@@ -0,0 +1,103 @@
+
+{{< figure src="../images/4.4.0-8391-cht-dialogs.png" class="left col-12 col-lg-12" >}}
+<br clear="all">
+
+- [#8390](https://github.com/medic/cht-core/issues/8390): Align "dialog" components with Material design guidelines
+
+
+## Highlights
+
+### Modals\'s text
+
+Some modals\'s text have been improved to increase understanding and some keys have been replaced.
+
+- Bulk delete reports
+  - `bulkdelete.confirm.title` used for the modal title when deleting 1 report
+  - [NEW] `bulkdelete.confirm.title.plural` used for the modal title when deleting multiple reports.
+- Edit message group
+  - [NEW] `edit_message_group.modal.title`: used for the modal title
+- Report a bug
+  - [NEW] `feedback.modal.submit`: used for the modal submit button label. 
+- Confirm Navigation - like when exiting a form that hasn't been submitted
+  - [NEW] `confirm.destructive.navigation.title`: used for the modal title
+  - [NEW] `confirm.destructive.navigation.submit`: used for the modal submit button label.
+- Training cards
+  - [NEW] `training_cards.confirm.title`: used for the modal title
+- Confirm verifying a report
+  - [NEW] `confirm.verification.title`: used for the modal title
+  - [NEW] `confirm.verification.submit`: used for the modal submit button label.
+
+We have provided translations for all of these in the CHT Core supported languages (English, Spanish, French, Indonesian, Hindi, Nepali, Bambara, Swahili)
+
+### Features
+
+None.
+
+### Improvements
+
+- [#6514](https://github.com/medic/cht-core/issues/6514): Migrate from SW-Precache to Workbox
+- [#8179](https://github.com/medic/cht-core/issues/8179): nginx and haproxy should respect `Accept` headers when responding with error templates
+- [#8390](https://github.com/medic/cht-core/issues/8390): Align "dialog" components with Material design guidelines
+- [#8442](https://github.com/medic/cht-core/issues/8442): Telemetry geolocation:failure events for geolocation should be unique
+- [#8498](https://github.com/medic/cht-core/issues/8498): Remove targets disable admin option
+- [#8506](https://github.com/medic/cht-core/issues/8506): Update browser support notice to match official versions
+- [#8511](https://github.com/medic/cht-core/issues/8511): Update configuration deployment action to use latest cht-conf
+
+### Security fixes
+
+- [#8443](https://github.com/medic/cht-core/issues/8443): Security issue 8156
+
+### Performance improvements
+
+- [#8238](https://github.com/medic/cht-core/issues/8238): Figure out a different way to use purged docs cache
+- [#8451](https://github.com/medic/cht-core/issues/8451): Add support for HTTP/2 in nginx 
+
+### Bug fixes
+
+- [#6180](https://github.com/medic/cht-core/issues/6180): Any text containing back ticks looks weird in the webapp
+- [#8081](https://github.com/medic/cht-core/issues/8081): `/api/v2/export/contacts` does not return useful contact type data for custom hierarchy
```

Comment:
I think here: `Add contact_type and place_id to contacts exported by /api/v2/export/contacts`